erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -3,7 +3,7 @@
  "allow_guest_to_view": 0, 
  "allow_import": 1, 
  "allow_rename": 0, 
- "autoname": "naming_series:", 
+ "autoname": "Prompt", 
```

Comment:
Why Prompt?

Reply:
Strange, I did not do that, let me fix it.